app/lucy/src/app/services/router.service.ts

```diff
@@ -168,4 +177,26 @@ export class RouterService {
     }
     return undefined;
```

Comment:
just return ... the js run time will consider it as undefined, specifying undefined is not require